### PR TITLE
fix storageVolumes demo config error

### PR DESCRIPTION
**Pull request by mikechengwei**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

Fix storageVolumes demo yaml configuration.

### CODE REVIEW

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
     config:
       log:
         file:
-          name: /var/log/tidblog
+          filename: /var/log/tidblog
```
Comment (mahjonp) [inline zh/configure-a-tidb-cluster.md:119]: cannot use a directory as the log file, how about using "/var/log/tidblog/tidb.log"

File: zh/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
     config:
       log:
         file:
-          name: /var/log/pd
+          filename: /var/log/pd
```
Comment (mahjonp) [inline zh/configure-a-tidb-cluster.md:105]: ditto

File: zh/configure-a-tidb-cluster.md (hunk 1 above)
Comment (mikechengwei) [reply zh/configure-a-tidb-cluster.md:119]: done

**Review by mahjonp — APPROVED** [review]
lgtm

**Review by ran-huang — APPROVED** [review]

**Pull request merged**